op-challenger/fault/position.go

```diff
@@ -49,6 +49,14 @@ func boolToInt(b bool) int {
 	}
 }
 
+// GetParentGIndex returns the gindex of the parent.
+func (p *Position) GetParentGIndex() uint64 {
```

Comment:
ParentGIndex is not the same as the gindex of the node that we are countering when we defend (only when we attack).